tst/testinstall/cset.tst

```diff
@@ -0,0 +1,43 @@
+#############################################################################
+##
+##  test of group intersection and RightCoset
+##
+gap> START_TEST("cset.tst");
+gap> Intersection(SymmetricGroup([1..5]),SymmetricGroup([3..8]));
+Sym( [ 3 .. 5 ] )
+gap> Intersection(SymmetricGroup([1..5]),AlternatingGroup([3..8]));
+Alt( [ 3 .. 5 ] )
+gap> Intersection(AlternatingGroup([1..5]),AlternatingGroup([3..8]));
+Alt( [ 3 .. 5 ] )
+gap> Intersection(AlternatingGroup([1..5]),SymmetricGroup([3..8]));
+Alt( [ 3 .. 5 ] )
+gap> Intersection(Group( (1,2,3), (4,5,6), (11,12,13), (11,12,14) ),
+                  Group( (1,7,8), (4,9,11), (10,12), (13,14) ));
```

Comment:
That's not valid syntax for a `.tst` file, and hence they fail in CI.
``suggestion
>                 Group( (1,7,8), (4,9,11), (10,12), (13,14) ));
```

Have you tested them locally? Assuming you started GAP from the root of the git repo, you can do so via `Test("tst/testinstall/cset.tst");`. As an advanced feature, you can use `Test("tst/testinstall/cset.tst", rec(rewriteToFile:=true));` to update the .tst file (this will update the "outputs" in the .tst file to match whatever GAP actually outputs when reading the "inputs". So this is not good to deal with issues in the inputs, as here, but it can be helpful when the outputs need tweaking)